Also note that the build fails:
```
/source/eval/src/tests/ann/xp-hnsw-wrap.cpp:5:10: fatal error: /git/hnswlib/hnswlib/hnswlib.h: No such file or directory
 #include "/git/hnswlib/hnswlib/hnswlib.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/eval/src/tests/ann/CMakeLists.txt
+++ b/eval/src/tests/ann/CMakeLists.txt
@@ -5,7 +5,6 @@ vespa_add_executable(eval_sift_benchmark_app
     sift_benchmark.cpp
     xp-annoy-nns.cpp
     xp-hnswlike-nns.cpp
-    xp-hnsw-wrap.cpp
     xp-lsh-nns.cpp
     DEPENDS
     vespaeval
@@ -16,7 +15,6 @@ vespa_add_executable(eval_remove_bm_app
     remove-bm.cpp
     xp-annoy-nns.cpp
     xp-hnswlike-nns.cpp
-    xp-hnsw-wrap.cpp
     xp-lsh-nns.cpp
     DEPENDS
     vespaeval

--- a/eval/src/tests/ann/remove-bm.cpp
+++ b/eval/src/tests/ann/remove-bm.cpp
@@ -1,4 +1,4 @@
-// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include <vespa/vespalib/testkit/test_kit.h>
 #include <vespa/vespalib/util/priority_queue.h>

